GalacticWasteManagement.Tests/Scripts/ScriptUtilitiesTests.cs

```diff
@@ -18,15 +14,15 @@ public void SplitInBatches_HappyDays()
             var somethingToSplit = @"SELECT * FROM Table1
               GO
               SELECT * FROM Table2";
-            var actual = ScriptUtilities.SplitInBatches(somethingToSplit);
+            var actual = new SqlServer.MsSql120ScriptParser().SplitInBatches(somethingToSplit);
```

Comment:
Looks like these tests no longer test our own code. Can we rephrase them? Otherwise I guess we could remove them?

Reply:
Depends on what you mean... it's still our code in a sense...

Reply:
But maybe the class shouldn't be named `ScriptUtilitiesTests` ...anymore

Reply:
Ah ok I though it was Microsofts code. Then I am fine with it!